src/jit/SimdX86Inl.h

```diff
@@ -903,12 +909,24 @@ static void emitUnarySIMD(sljit_compiler* compiler, Instruction* instr)
     case ByteCode::I32X4TruncSatF32X4UOpcode:
         simdEmitTruncSatF32x4U(compiler, dst, args[0].arg, instr->requiredReg(1));
         break;
+    case ByteCode::I32X4RelaxedTruncF32X4SOpcode:
+        simdEmitSSEOp(compiler, SimdOp::cvttps2dq, dst, args[0].arg);
+        break;
+    case ByteCode::I32X4RelaxedTruncF32X4UOpcode:
+        simdEmitTruncSatF32x4U(compiler, dst, args[0].arg, instr->requiredReg(1));
```

Comment:
I could not simplify it, since x86 has no unsigned conversion. There might some simplification if somebody can figure that out.